frontend/src/components/ProjectItem/index.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import DeleteIcon from '@mui/icons-material/Delete';
+import IconButton from '@mui/material/IconButton';
+import EditIcon from '@mui/icons-material/Edit';
+import ListItemButton from '@mui/material/ListItemButton';
+import Typography from '@mui/material/Typography';
+import Project from '../../types/Project';
+import ListItemText from '@mui/material/ListItemText';
+import LinearProgress, { LinearProgressProps } from '@mui/material/LinearProgress';
+import { Box, ListItem, ListItemIcon, ListItemSecondaryAction } from '@mui/material';
+import Grid from '@mui/material/Unstable_Grid2';
+
+interface GetTaskFormProps {
+  project: Project;
+}
+
+function LinearProgressWithLabel(props: LinearProgressProps & { value: number }) {
+  return (
+    <Box sx={{ display: 'flex', alignItems: 'center' }}>
+      <Box sx={{ width: '100%', mr: 1 }}>
+        <LinearProgress variant="determinate" {...props} />
+      </Box>
+      <Box sx={{ minWidth: 35 }}>
+        <Typography variant="body2" color="text.secondary">{`${Math.round(
+          props.value
+        )}%`}</Typography>
+      </Box>
+    </Box>
+  );
+}
+
+export default function TaskListItem(props: GetTaskFormProps) {
+  return (
+    <ListItem disablePadding>
+      <ListItemButton>
+        <ListItemText disableTypography>
+          <Grid container spacing={2} alignItems="center" justifyContent="space-between">
+            <Grid xs="auto" sx={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
just realized this will cause the progress bars to not be aligned. change the `xs="auto"` to `xs={2}` and then ill create a fix pr to do the same with the task component